playbooks/cdh-web.yml

```diff
@@ -1,21 +1,19 @@
 - hosts: cdhweb_prod
   connection: ssh
-  remote_user: deploy
-  # Set environment to use scl rh-python35 in group_vars
-  environment:
-    PATH: '{{ path }}'
-    LD_LIBRARY_PATH: '{{ ld_library_path }}'
-    PYTHONPATH: '{{ python_path }}'
+  remote_user: pulsys
   roles:
     - create_deployment
+    - deploy_user
     - build_project_repo
     - build_virtualenv
+    - postgresql
+    - configure_logging
+    - configure_apache
     - install_local_settings
     - build_npm
     - configure_media
     - django_collectstatic
     - django_compressor
-    - backup_database
```

Comment:
Why are we dropping this? Does it not work with postgres?

I would still like to get a snapshot before running migration if we can. (I know we discussed this and I know postgres has a lot more capability than we had before but I don't remember where we ended!)

Reply:
This might've just been temporarily disabled because of a permissions issue I was having or something, good catch. I'll add it back

Reply:
ah, I keep forgetting! `backup_database` is now a springdale-only role. the `postgresql` role handles backing up with one of its tasks, so no need to include `backup_database` for any PUL-hosted projects.